keyboards/qwertykeys/qk80/ansi/readme.md

```diff
@@ -0,0 +1,21 @@
+# qk80 - ANSI PCB
+
+![qk80](https://i.imgur.com/qSXHoWyh.jpg)
```

Comment:
this image is of a tsangan bottom row layout rather than standard TKL bottom row layout, could this be amended if possible?